subprojects/packagefiles/taskflow/meson.build

```diff
@@ -0,0 +1,13 @@
+project('taskflow', 'cpp',
+    version: 'v2.7.0',
+)
+
+# Header-only library
+taskflow_inc = include_directories('.')
+
+taskflow_dep = declare_dependency(
+    include_directories: taskflow_inc,
+    dependencies: [
+        dependency('threads'),
+    ]
+)
```

Comment:
no newline?

Reply:
Newline added in the new change. Thank you.